rir/src/compiler/opt/delay_env.h

```diff
@@ -18,7 +18,7 @@ class DelayEnv : public PirTranslator {
   public:
     DelayEnv() : PirTranslator("Delay Environment"){};
 
-    void apply(Closure* function) override;
+    void apply(Closure* function) const final;
```

Comment:
Isn't it possible to keep the override?

Reply:
Same for all the applies...

Reply:
I am wrong, you are right. (https://en.cppreference.com/w/cpp/language/final)